woocommerce/woocommerce
# update woo-gutenberg-products-block text domain to woocommerce in src folder

### Changes proposed in this Pull Request:

This PR updates the text domains in the `plugins/woocommerce/src` folder from `woo-gutenberg-products-block` to `woocommerce`.

See #41996 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Review text domain changes
2. Check that a string wasn't inadvertently changed
3. check locally that the blocvks text domain isn't found in any files
```
cd src
grep -nR woo-gutenberg ./
```



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
